packages/jsii-rosetta/bin/jsii-rosetta.ts

```diff
@@ -10,7 +10,7 @@ import { readTablet as readTablet } from '../lib/commands/read';
 import { translateMarkdown } from '../lib/commands/convert';
 import { File, printDiagnostics, isErrorDiagnostic } from '../lib/util';
 
-async function main() {
```

Comment:
what's up here? why is this not async anymore?

Reply:
It does not return a `Promise` so it may not be async. It returns a plain `void`.